spec/integration/metric_spec.rb

```diff
@@ -70,6 +70,25 @@
   end
   #  end
 
+  it 'Should requests raw data for multiple metrics' do
+    @client = setup_client(username: 'jdoe', password: 'password')
+    ids = [SecureRandom.uuid, SecureRandom.uuid, SecureRandom.uuid]
+    VCR.use_cassette('Mixed_metrics/Should requests raw data for multiple metrics',
+                     erb: { ids: ids }, record: :none
+                    ) do
+      expect(@client.avail.raw_data(ids).size).to be 0
+
+      @client.push_data(
+        availabilities: [
+          { id: ids[0], data: [{ value: 'up' }] },
+          { id: ids[1], data: [{ value: 'down' }] },
+          { id: ids[2], data: [{ value: 'up' }] }
+        ]
+      )
+      expect(@client.avail.raw_data(ids).size).to be 3
```

Comment:
Can you please explicitly test that the right datapoints in the expected format are returned?